mrtarget/common/ElasticsearchLoader.py

```diff
@@ -187,8 +187,8 @@ def _safe_create_index(self, index_name, body={}, ignore=400):
         if not self.dry_run:
             res = self.es.indices.create(index=index_name, ignore=ignore, body=body )
             if not self._check_is_aknowledge(res):
-                if res['error']['root_cause'][0]['reason']== 'already exists':
-                    self.logger.error('cannot create index %s because it already exists'%index_name) #TODO: remove this temporary workaround, and fail if the index exists
+                if 'already exists' in res['error']['root_cause'][0]['reason']:
+                    self.logger.warning('cannot create index %s because it already exists'%index_name) #TODO: move the creation of index to the main thread to avoid all threads trying this
```

Comment:
so the previous comments states that it should actually _fail_ if the index already exists. This is not what my code is doing now. Should it be adjusted to _fail_? So raising an exception instead of logging a warning?